src/import-service/src/functions/utils/importUtils.ts

```diff
@@ -0,0 +1,107 @@
+import * as AWS from "aws-sdk";
+const csv = require("csv-parser");
+
+const getS3BucketInstance = () => {
```

Comment:
You loose in performance as well as money when you create new Sdk Client for every new function.
Better reuse this logic by defining all the clients beforehand.

https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html
Take advantage of execution environment reuse to improve the performance of your function. Initialize SDK clients and database connections outside of the function handler, and cache static assets locally in the /tmp directory. Subsequent invocations processed by the same instance of your function can reuse these resources. This saves cost by reducing function run time.

Reply:
Will change this.